WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -249,8 +252,26 @@ private void loggedInAndFinish(ArrayList<Integer> oldSitesIds, boolean doLoginUp
                 break;
             case SHARE_INTENT:
             case SELFHOSTED_ONLY:
+                // We are comparing list of site ID's before self-hosted site was added and after, trying to find a
+                // newly added self-hosted site's ID, so we can select it
+                ArrayList<Integer> currentSiteIds = new ArrayList<>();
```

Comment:
Nit that I realized just now, what do you think about renaming this variable to `newSitesIds` instead of `current...` since its purpose is to hold the IDs of the new sites?